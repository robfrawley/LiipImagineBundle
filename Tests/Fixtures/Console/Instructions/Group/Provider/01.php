<?php

/*
 * This file is part of the `liip/LiipImagineBundle` project.
 *
 * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
 *
 * For the full copyright and license information, please view the LICENSE.md
 * file that was distributed with this source code.
 */

use Liip\ImagineBundle\Console\Output\Markup;
use Liip\ImagineBundle\Console\Output\MarkupColors;
use Liip\ImagineBundle\Console\Style\Style;
use Liip\ImagineBundle\Tests\Fixtures\Console\FixtureProvider;

return function (Style $style, FixtureProvider $fixture) {
    foreach ($fixture->data('group', 'items') as $item => $group) {
        $style
            ->group($item, $group)
            ->newline(2)
            ->group($item, $group, new Markup(MarkupColors::COLOR_BLUE))
            ->newline(2)
            ->group($item, $group, new Markup(MarkupColors::COLOR_WHITE, MarkupColors::COLOR_WHITE))
            ->newline(2);
    }
};
