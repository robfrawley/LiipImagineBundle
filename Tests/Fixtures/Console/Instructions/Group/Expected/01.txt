<options=bold>item-01.ext[</>group-namespace-name<options=bold>]</>

<fg=blue;options=bold>item-01.ext[</><fg=blue>group-namespace-name</><fg=blue;options=bold>]</>

<fg=white;bg=white;options=bold>item-01.ext[</><fg=white;bg=white>group-namespace-name</><fg=white;bg=white;options=bold>]</>

<options=bold>item-02.ext[</>group-namespace-name<options=bold>]</>

<fg=blue;options=bold>item-02.ext[</><fg=blue>group-namespace-name</><fg=blue;options=bold>]</>

<fg=white;bg=white;options=bold>item-02.ext[</><fg=white;bg=white>group-namespace-name</><fg=white;bg=white;options=bold>]</>

<options=bold>item-03.ext[</>group-namespace-name<options=bold>]</>

<fg=blue;options=bold>item-03.ext[</><fg=blue>group-namespace-name</><fg=blue;options=bold>]</>

<fg=white;bg=white;options=bold>item-03.ext[</><fg=white;bg=white>group-namespace-name</><fg=white;bg=white;options=bold>]</>

<options=bold>item-04.ext[</>group-namespace-name<options=bold>]</>

<fg=blue;options=bold>item-04.ext[</><fg=blue>group-namespace-name</><fg=blue;options=bold>]</>

<fg=white;bg=white;options=bold>item-04.ext[</><fg=white;bg=white>group-namespace-name</><fg=white;bg=white;options=bold>]</>

<options=bold>item-05.ext[</>group-namespace-name<options=bold>]</>

<fg=blue;options=bold>item-05.ext[</><fg=blue>group-namespace-name</><fg=blue;options=bold>]</>

<fg=white;bg=white;options=bold>item-05.ext[</><fg=white;bg=white>group-namespace-name</><fg=white;bg=white;options=bold>]</>

